.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,110 @@
+name: "\U0001F41B Bug Report"
+description: "If something isn't working as expected \U0001F914"
+labels: ["needs triage"]
+body:
+    - type: markdown
+      attributes:
+          value: |
+              ## :warning: We use GitHub Issues to track bug reports and feature requests
+
+              If you are not sure that your issue is a bug, you could:
+
+              - use our [Discord community](https://discord.gg/jXcMagAPnm)
+              - use [StackOverflow using the tag `inversifyjs`](https://stackoverflow.com/questions/tagged/inversifyjs)
+
+              **NOTE:** You don't need to answer questions that you know that aren't relevant.
+
+              ---
+
+    - type: checkboxes
+      attributes:
+          label: "Is there an existing issue for this?"
+          description: "Please search [here](https://github.com/inversify/InversifyJS/issues) to see if an issue already exists for the bug you encountered"
+          options:
+              - label: "I have searched the existing issues"
+                required: true
+
+    - type: textarea
+      validations:
+          required: true
+      attributes:
+          label: "Current behavior"
+          description: "How the issue manifests?"
+
+    - type: textarea
+      attributes:
+          label: "Steps to reproduce"
+          description: |
+              How the issue manifests?
+              You could leave this blank if you can't reproduce it, but please provide as much information as possible
+          placeholder: |
+              1. `npm ci`
+              2. `npm start:dev`
+              3. See error...
+
+    - type: textarea
+      validations:
+          required: true
+      attributes:
+          label: "Expected behavior"
+          description: "A clear and concise description of what you expected to happened (or code)"
+
+    - type: textarea
+      validations:
+          required: false
+      attributes:
+          label: "Possible solution"
+          description: "If you have a suggestion on how to fix the bug"
+
+    - type: markdown
+      attributes:
+          value: |
+              ---
+
+    - type: input
+      validations:
+          required: true
+      attributes:
+          label: "Package name"
+          description: "What package is affected by this issue?"
+          placeholder: "inversify"
```

Comment:
```suggestion
```
I'd say this repository is only handling `inversify` so we don't need this field